datatypes/src/commonMain/kotlin/at/asitplus/crypto/datatypes/CryptoPublicKey.kt

```diff
@@ -298,6 +300,6 @@ sealed class CryptoPublicKey : Asn1Encodable<Asn1Sequence>, Identifiable {
 }
 
 private fun sanitizeRsaInputs(n: ByteArray, e: Int) = n.dropWhile { it == 0.toByte() }.toByteArray()
-    .let { Triple(it, e, CryptoPublicKey.Rsa.Size.of(it)) }
+    .let { Triple(byteArrayOf(0, *it), e, CryptoPublicKey.Rsa.Size.of(it)) }
```

Comment:
What are the semantics of this triple?

Reply:
Come to think of it: Do we even want the SIZE parameter in the constructor? Should this not always be computed from `n` and never be passed?